README.md

```diff
@@ -39,8 +35,17 @@ Testing
 We currently use [travis-ci](https://travis-ci.org/xbmc/Kore/) to automatically build
 and run the local tests for each pull request.
 
-Credits
--------
+## Using Docker
+
+1. Make sure you have a working [Docker installation](https://docs.docker.com/docker-for-windows/install/);
+2. Check out the repository
+3. Build the container image: `docker build -t kore:latest .`
+4. Start container: `docker run -it -v $(pwd):/opt/kore kore:latest bash`
+
+For listing all tasks, run `gradle tasks`, for building the app, execute `gradle assembleRelease`.
+If you want to run tests, execute `gradle testDebugUnitTest`.
```

Comment:
I can remove these changes and the Dockerfile if you want. I just found it very useful for setting up a dev-environment in a container (and not having to fiddle with my local setup).

Reply:
I don't use it, but agree that it can be useful to some. Nevertheless, i can't compromise on keeping the docker configuration up to date, so i'll probably add a disclosure in the readme alerting to that.